Documentation/Developer/InterlinkInventories.rst

```diff
@@ -92,11 +96,61 @@ The following links:
 
 
 Adding a new TYPO3 version or manual
-====================================
+------------------------------------
 
 In the event of a change in long-term support, adjustments to the corresponding
 TYPO3 versions can be made directly in the theme within the enum
 :php:`\T3Docs\Typo3DocsTheme\Inventory\Typo3VersionMapping`.
 
 The default manuals to be supported can be managed in enum
 :php:`\T3Docs\Typo3DocsTheme\Inventory\DefaultInventories`.
+
+
+Interlinks to TYPO3 extensions
+==============================
+
+You can link to the main version of the manual of a TYPO3 extension if
+that extension's manual has been rendered on our server.
+
+To create an interlink to a third-party extension prefix the extensions
+composer name with `ext/` and use it as interlink domain:
+
+..  code-block:: rst
+
+    *   :doc:`News <ext/georgringer/news:Index>`
+    *   :ref:`External Imports <ext/cobweb/external_import:start>`
+
+This will be rendered as:
+
+*   :doc:`News <ext/georgringer/news:Index>`
+*   :ref:`External Imports <ext/cobweb/external_import:start>`
+
+
+If an extension author need to link to a specific version of an extensions manual,
+they have to define that version manually in the manual's :file:`guides.xml`:
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+    >
+        <project title="Render guides"/>
+        <extension
+            class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+            typo3-core-preferred="stable"
+        />
+        <!-- explicitly link to version 11.3 of the news manual -->
+        <inventory id="ext/georgringer-news-11.3" url="https://docs.typo3.org/p/georgringer/news/11.3/en-us/"/>
```

Comment:
Hmm, so sometimes you use `ext/` and sometimes `ext_` as id - so there is no rule?

Above, there is a path (I assume) on the server for the main branch, here we should stick to `ext_` like suggested today.

Reply:
ext_sys_notes is not a composer name. I would suggest to phase out the old keys and deprecate them a while after our switch. we can then use composer names for each and every extension, including core extensions

Reply:
But "ext/typo3/cms-sys-note" is.

Reply:
So what exactly do you suggest? Please let us discuss in only one place this gets confusing

Reply:
I like to use the same structure, not a mixture of `ext_`and `ext/`. So, when we go with `ext/` the sys_note should also be adjusted. Or is there a rule, when to use the one or the other I don't get?

Reply:
But, anyway, I don't care anymore about that. Do it, when you think that is the way.

Reply:
@brotkrueml I gave it another try to make it more logic. The ext prefix is not really needed as we didn't have slashes in the interlink domain in sphinx. So using the composer name as interlink domain now works for both third party extensions and system extensions. The legacy names ext_adminpanel etc are kept for now, we can deprecate them later. Also I automatically made the versions linkable. In almost all cases it should not be necessary anymore to define interlinks in the guides.

Reply:
I do not getting warm with the syntax. Once it is `t3coreapi/stable`, then it is `typo3/cms-sys-note/stable`. The second one is derived from the Composer package, the first one not. Personally, I would like to have at least another separator for the version, like: `t3coreapi@stable`  and `typo3/cms-sys-note@stable`. Sadly, we can't use the colon (like Composer) as this is already in use for a rst reference.

However, when you think this is the way, go ahead, I won't block it.

Reply:
I can investigate if we can use the colon syntax. We are overriding the interlink detection so if we get the regex worked out we can do anything we want to

Reply:
After thinking again, colon is more confusing. Think of: `typo3/cms-sys-note:stable:the-ref` That's not worth the work.

Reply:
For me it is okay for now. Let's get it in, then it can be adjusted if necessary.